docs/reference/elasticsearch/rest-apis/update-document.md

```diff
@@ -0,0 +1,264 @@
+---
+mapped_pages:
+  - https://www.elastic.co/guide/en/elasticsearch/reference/8.18/docs-update.html
+applies_to:
+  stack: all
+navigation_title: Update a document
+---
+
+# Update a document [update-document]
+
+The [Update API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-update) enables you to script document updates, allowing you to update, delete, or skip modifying a document. 
+
+The following examples show common use cases, such as incrementing a counter and adding or removing elements from a list, as well as how to:
+
+- [Update part of a document](#update-part-document)
+- [Detect noop updates](#detect-noop-updates)
+- [Upsert](#upsert)
+- [Doc as upsert](#scripted-upsert)
```

Comment:
```suggestion
- [Insert or update documents with upsert](#upsert)
- [Simplify upsert with doc_as_upsert](#doc-as-upsert)
```

This suggestion:

- Uses parallel structure for all items in the list (verb + object/outcome)
- Explain what happens rather than just naming parameter, makes scanning easier

Reply:
Would need to update the subheadings accordingly to match if accept :)